docs/install.md

```diff
@@ -236,6 +226,13 @@ alternative](https://hub.docker.com/r/linuxserver/code-server).
 
 You can install code-server via [Helm](https://github.com/cdr/code-server/blob/main/ci/helm-chart/README.md).
 
+## Windows
+
+We currently do not publish Windows releases (see [#1397](https://github.com/cdr/code-server/issues/1397)). We recommend installing code-server onto Windows with [`yarn` or
```

Comment:
I like that you cross-linked them like this :+1:

Reply:
very minor suggestion is to turn the "do not publish Windows releases" text into a link, I think it's more accessible and larger text is easier to press on a mobile device.

```suggestion
We currently [do not publish Windows releases](https://github.com/cdr/code-server/issues/1397). We recommend installing code-server onto Windows with [`yarn` or
```